loader.js

```diff
@@ -29,7 +29,11 @@
       await new Promise(async resolve => {
         if ("document" in self) {
           const script = document.createElement("script");
+          // #ifdef publicPath
+          script.src = // #put "'" + publicPath + "' + name.slice(1);"
```

Comment:
How about you set the `publicPath` option to `./` as a default then you don’t need the conditional here.

Reply:
I can do — though my rationale was that if people *aren't* using the option, then they get

```js
script.src = name;
```

whereas if `publicPath` was always declared then it would be

```js
script.src = '.' + name.slice(1);
```

which is redundant. Or are you saying the logic should go in the `#put`?

Reply:
Hm, fair point. I guess in the name of file size we should prefer the build time complexity over runtime complexity. 

Okay, let’s do this.